examples/training/classification/basic_imagenet_training.json

```diff
@@ -0,0 +1,25 @@
+{
+    "script_authors": ["ianjjohnson"]
+    "densenet121": {
```

Comment:
so every model will land here basically?

Reply:
Right -- every model trained using this script.